src/lib/mina_compile_config/mina_compile_config.ml

```diff
@@ -67,44 +55,14 @@ let max_event_elements = 100
 
 let max_action_elements = 100
 
-[%%inject "network_id", network]
-
-[%%ifndef zkapp_cmd_limit]
```

Comment:
These kinds of conditional assignments are all pushed into the `Node_config` module